v19.1/performance-best-practices-overview.md

```diff
@@ -158,7 +158,7 @@ However, the performance best practice is to use a `RETURNING` clause with `INSE
 
 ### Use secondary indexes
 
-You can use secondary indexes to improve the performance of queries using columns not in a table's primary key. You can create them:
+You can use secondary [indexes](https://www.cockroachlabs.com/docs/stable/indexes.html) to improve the performance of queries using columns not in a table's primary key. You can create them:
```

Comment:
This is a good change, but we use relative links. I'll handle this in a separate PR.